src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -2026,31 +2054,28 @@
             <Style x:Key="IncreaseButton"
                    BasedOn="{x:Null}"
                    TargetType="{x:Type RepeatButton}">
-                <Setter Property="MinWidth" Value="10" />
+                <Setter Property="MinWidth" Value="12" />
                 <Setter Property="Template">
                     <Setter.Value>
                         <ControlTemplate TargetType="{x:Type RepeatButton}">
                             <Grid>
                                 <Border Name="Border"
                                         Margin="0"
                                         Padding="0"
-                                        Background="#3C3C3C"
-                                        BorderThickness="0" />
-                                <ContentPresenter x:Name="ContentPresenter"
-                                                  HorizontalAlignment="Center"
-                                                  VerticalAlignment="Center"
-                                                  Content="{TemplateBinding Content}"
-                                                  ContentTemplate="{TemplateBinding ContentTemplate}"
-                                                  ContentTemplateSelector="{TemplateBinding ContentTemplateSelector}" />
+                                        Background="{TemplateBinding Background}"
+                                        BorderThickness="0">
+                                    <Path Margin="1,1,0,0"
+                                          HorizontalAlignment="Center"
+                                          VerticalAlignment="Center"
+                                          Data="M-2.5,1 L0,-2 L2.5,1"
+                                          Fill="{x:Null}"
```

Comment:
Given this is null, do we have to set this?

Reply:
Seems to work fine without; have removed.